bin/memos/main.go

```diff
@@ -110,6 +111,7 @@ func init() {
 	rootCmd.PersistentFlags().String("driver", "sqlite", "database driver")
 	rootCmd.PersistentFlags().String("dsn", "", "database source name(aka. DSN)")
 	rootCmd.PersistentFlags().String("instance-url", "", "the url of your memos instance")
+	rootCmd.PersistentFlags().String("base-url", "", "the base url of your memos instance")
```

Comment:
Please add details to readme, how can this be configured and what should be used and when.

Reply:
> Please add details to readme, how can this be configured and what should be used and when.

By default, the deployment is at the root and does not require any modifications.
If you want to deploy memos to a subdirectory, you need to configure base-url (via environment variables or parameters when executing memos). 
For example, if you want to deploy memos in the /memos subdirectory of the domain name, run the following command:
MEMOS_MODE="prod" MEMOS_PORT=5230 /usr/local/bin/memos --base-url /memos
or
MEMOS_MODE="prod" MEMOS_PORT=5230 MEMOS_BASE_URL=/memos /usr/local/bin/memos

Of course, there is a slight requirement when compiling the memos frontend, you need to execute the sed command to adjust the index.html of the frontend after compiling the frontend (the corresponding command is already included in the Makefile).

`
	sed -i "s|<script type=\"module\" crossorigin src=\"|&{{ .baseurl }}|g"   ./web/dist/index.html
	sed -i "s|<link rel=\"stylesheet\" crossorigin href=\"|&{{ .baseurl }}|g" ./web/dist/index.html
`

Reply:
Please add this directly to documentation and/or readme so it's actually visible to users. Ideally with examples.in docker-compose or so

Reply:
> Please add this directly to documentation and/or readme so it's actually visible to users. Ideally with examples.in docker-compose or so

done.
add some info into readme.md